specification/schemas/HookLog.json

```diff
@@ -0,0 +1,121 @@
+{
+  "type": "object",
+  "required": [
+    "attributes",
```

Comment:
You can add "id" and "links" in here since they will always be there. We will never add a post hooklogs endpoint so there will never be a case that we will not have an id or links for this resource.

See above comment.